doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+Where the work is done e.g. work done on DKRZ/Levante:
+
+- submit files in `/home/b/b382109/submit` (for v2.7.0)
+- output in `/scratch/b/b382109/esmvaltool_output` (for v2.7.0)
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented:
+
+.. code-block:: bash
+
+  mamba env create -n tool2xxTest -f environment.yml
+  conda activate tool2xxTest
+
+Make a copy of the environment file, and attach it in the release testing issue; to
+record the environment in a yaml file use e.g. `conda env export > ToolEnv270Test.txt`.
+
+Mention any special data movement one needs to perfom to run the tests e.g.:
+for v2.7.0 the release manager moved the autoassess-specific files to
+`/home/b/$USER/autoassess_files` on DKRZ/Levante.
+
+Modifications to configuration files need to be documented as well: for example,
+for v2.7.0 the release manager had to modify `config-user.yml` file by
+adding a number of extra paths for DKRZ/Levante-specific data pools:
+
+.. code-block:: yaml
+
+  CMIP6:
+    - /work/bd0854/DATA/ESMValTool2/CMIP6_DKRZ
+    - /work/bd0854/DATA/ESMValTool2/download/CMIP6
+  CMIP5:
+    - /work/bd0854/DATA/ESMValTool2/CMIP5_DKRZ
+    - /work/bd0854/b309141/additional_CMIP5
+    - /work/bd0854/DATA/ESMValTool2/download/cmip5/output1
+    - /work/bd0854/DATA/ESMValTool2/download/cmip5
+
+Submit run scripts - test recipe runs
+-------------------------------------
+
+We are now ready to start running all the available recipes, to compare output against previous release. Running is currently done
+via batch scripts submitted to a schedulers (SLURM). Generate the submission scripts using the `generate.py` Python script;
+you can find a copy of the script either in `/home/b/b382109/Tool_Release_270_Scripts` or
+in the draft Pull Request https://github.com/ESMValGroup/ESMValTool/pull/2883.
+
+You will have to set the name of your environment, your email address (if you want to get email notifications for successful/failed jobs) and the name of the directory you want to store the job outputs. The name of the account is the same (`bk1088`), and the default partition is set to `compute`.
```

Comment:
I wonder if we can still use that account after IS-ENES3 ends

Reply:
no clue - we need to ask, I can email DKRZ?

Reply:
I just plopped a note there, more for us to remember to check, then we can change the note to a warning

Reply:
I can take care of that. I think I know who to ask, I'll cc you